src/main/java/com/cairn/waypoint/dashboard/service/helper/HomeworkQuestionResponseHelperService.java

```diff
@@ -0,0 +1,24 @@
+package com.cairn.waypoint.dashboard.service.helper;
+import org.springframework.stereotype.Service;
+
+import com.cairn.waypoint.dashboard.endpoints.homework.dto.UpdateHomeworkResponseDetailsDto;
+import com.cairn.waypoint.dashboard.entity.HomeworkResponse;
+import com.cairn.waypoint.dashboard.service.data.ExpectedResponseDataService;
+
+@Service
+@SuppressWarnings("DuplicatedCode")
```

Comment:
If there is duplicated code, try to refactor so that you don't need to suppress it. This could be discretionary but we can discuss